bin/desi_create_tiletable

```diff
@@ -109,6 +109,7 @@ def main():
     import argparse
     p = argparse.ArgumentParser()
     p.add_argument('-o', '--outfile', required=True, help="output filename")
+    p.add_argument('--outfile2', required=False, help="Additional output filename (e.g. csv instead of fits)")
```

Comment:
This second outfile should not be necessary. The script should have enough logic to know that it needs to create two files, and if `outfile` is *.csv it needs to create *.fits and vice versa.

Reply:
I updated the logic to auto-derive the alternate extension.

Reply:
Thank you!